server/src/project/project.service.ts

```diff
@@ -395,6 +411,23 @@ export class ProjectService {
 
     transformedProject.video_links = await getVideoLinks(this.config.get('AIRTABLE_API_KEY'), firstProject.dcp_name);
 
+    let { records: projectPackages } = await this.crmService.get('dcp_packages', `
+        $filter=
+          _dcp_project_value eq ${firstProject.dcp_projectid}
+          and (
+            dcp_visibility eq ${PACKAGE_VISIBILITY.GENERAL_PUBLIC}
+          )
+          and (
+            statuscode eq ${PACKAGE_STATUSCODE.SUBMITTED}
+          )
+      `);
+
+    projectPackages = await Promise.all(projectPackages.map(async (pkg) => {
+        return await this.documentService.packageWithDocuments(pkg);
```

Comment:
is there a way to double check that certian packages should be making this query? perhaps we could check that the pkg is public? that would spare us some network requests.

Reply:
yeah, the code above (414 - 423) limits the sideloaded packages to only those that are public and submitted. Other packages are not included here.